movingSquare.js

```diff
@@ -0,0 +1,93 @@
+const square = document.createElement('div')
+const h6 = document.createElement('h6')
+
+let positionX = 0
+let positionY = 0
+
+square.style.backgroundColor = 'red'
```

Comment:
It is ok, but it does not look good that you just code line by line without wrapping parts of your code in functions.

This should go into function name, e.g. `createSquare` that will return ready to use DOM object.